src/components/ui/ResultCount.js

```diff
@@ -25,7 +25,7 @@ const ResultCount = ({ items }: Props) => {
   const resultText = parseInt(itemsCount, 10) !== 1 ? 'results' : 'result'
 
   return (
-    <span className={cx.wrapper}>
+    <span className={cx.wrapper} data-testid='xpui-result-counter'>
```

Comment:
Just a suggestion, to keep it consistent with other changes, maybe this should be:

```suggestion
    <span className={cx.wrapper} data-testid='xpui-resultCount'>
```

Reply:
@kararade can you check if this suggestion makes sense? Just to confirm, if something not needed we can mark as resolved. Thanks